pkgs/tools/misc/git-town/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }: 
+
+let
+  version = "7.2.0";
+in
+  buildGoPackage rec {
+    name = "git-town-${version}";
+    version = "7.2.0";
+
+    goPackagePath = "github.com/Originate/git-town";
+
+    src = fetchFromGitHub {
+      owner = "Originate";
+      repo = "git-town";
+      rev = "v${version}";
+      sha256 = "0hr0c6iya34lanfhsg9kj03l4ajalcfxkbn4bgwh0749smhi6mrj";
+    };
+
+    buildFlagsArray = ''
```

Comment:
buildFlagsArray should be a list of strings, not a string I guess

Reply:
Fixed